fault/assert_immediate.py

```diff
@@ -33,6 +34,7 @@ def assert_immediate(cond, success_msg=None, failure_msg=None, severity="error",
         on = on.compile(format_args)
     else:
         raise TypeError(f"Unexpected type for on={type(on)}")
+    name_str = "" if name is None else f"{name}: "
     m.inline_verilog(
-        "always {on} assert ({cond}){success_msg_str}{failure_msg_str};",
+        "always {on} {name_str}assert ({cond}){success_msg_str}{failure_msg_str};",
```

Comment:
need space before `assert`?

Reply:
`name_str` includes a space after it if it's present, so no space here (in the case where it's empty string, we avoid thee extra space)